test/e2e/topology_updater/topology_updater.go

```diff
@@ -96,6 +97,33 @@ var _ = ginkgo.Describe("[TopologyUpdater][InfraConsuming] Node topology updater
 	})
 
 	ginkgo.Context("[release] with cluster configured", func() {
+		ginkgo.It("should have Node as Owner Reference", func() {
+			ginkgo.By("getting the initial topology information")
+			expectedOwnerReference := metav1.OwnerReference{
+				Name:       topologyUpdaterNode.Name,
+				APIVersion: topologyUpdaterNode.APIVersion,
+				Kind:       topologyUpdaterNode.Kind,
+				UID:        topologyUpdaterNode.UID,
+			}
+			gomega.Eventually(func() bool {
+				finalNodeTopo, err := f.TopoCli.TopologyV1alpha2().NodeResourceTopologies().Get(context.TODO(), topologyUpdaterNode.Name, metav1.GetOptions{})
+				if err != nil {
+					klog.Infof("failed to get the node topology resource: %v", err)
+					return false
+				}
+
+				if len(finalNodeTopo.OwnerReferences) == 1 {
+					klog.Infof("Too many OwnerReferences: expected=1; got=%d", len(finalNodeTopo.OwnerReferences))
+					klog.Infof("%+#v", finalNodeTopo.OwnerReferences)
+					return false
+				}
+				if !reflect.DeepEqual(finalNodeTopo.OwnerReferences[0], expectedOwnerReference) {
+					klog.Infof("Unexpected OwnerReference: expected=%+#v; got=%+#v", expectedOwnerReference, finalNodeTopo.OwnerReferences[0])
+					return false
+				}
+				return true
+			}, 5*timeout, 5*time.Second).Should(gomega.BeTrue(), "didn't get updated node topology info")
```

Comment:
please let's use `WithPolling` and `WithTimeout` in new code like in https://github.com/k8stopologyawareschedwg/resource-topology-exporter/blob/main/test/e2e/rte/rte.go#L152
(yes, we should send a followup PR later on to modernize the usages)

Reply:
https://github.com/k8stopologyawareschedwg/resource-topology-exporter/pull/214

Reply:
done, Thanks for the comment.